att.go

```diff
@@ -0,0 +1,188 @@
+//go:build nina || nano_rp2040
+
+package bluetooth
+
```

Comment:
Because this file is specific to the NINA, I think this file should be named something like att_nina.go.

Reply:
Done.